CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 SDK Changelog
 =============
 
+1.6.0 (2023-10-??)
+==================
+
+.. warning:: SDK 1.5.0 requires at least version 2.4.0 of the Open Formulieren API.
```

Comment:
Just checking: shouldn't it be 1.6? :thinking:

Reply:
oops! I updated it